libs/ui/src/hooks/smartcard_auth/auth_helpers.test.ts

```diff
@@ -599,6 +617,43 @@ describe('Card interface', () => {
     spy.mockRestore();
     expect((await card.unprogramUser()).isOk()).toEqual(true);
   });
+
+  it('can program and unprogram cards without a logger', async () => {
```

Comment:
Needed to add this test to get back up to 100% code coverage - this separately made me question whether we should even allow instantiating the smartcard auth hooks without loggers, when logging's a VVSG requirement. But that's something to revisit in a separate PR!